src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedOptions.kt

```diff
@@ -50,6 +52,13 @@ abstract class ManagedOptions<OPT : Options<OPT, CTEST>, CTEST : CTestConfigurat
         this.hangingDetectionThreshold = hangingDetectionThreshold
     }
 
+    /**
+     * Set to `true` to activate the static memory snapshot tracking and restoring algorithm.
+     */
+    fun restoreStaticMemory(restoreStaticMemory: Boolean): OPT = applyAndCast {
+        this.restoreStaticMemory = restoreStaticMemory
+    }
```

Comment:
I thought it was decided to remove this parameter?

Reply:
done